jaggr-sample/WebContent/js/LazyContentPane.js

```diff
@@ -42,10 +43,10 @@ define([
 					}));
 				} else if (title == 'Editor') {
 					require(['dijit/Editor'], lang.hitch(this, function(editor) {
-						this.set('content', '<div></div>');
-						new editor({
+						dconst.empty(this.domNode);

```

Comment:
The loading spinner.  The previous implemented reset the content with a set call.